src/pycolmap/estimators/similarity_transform.cc

```diff
@@ -54,8 +54,9 @@ void BindSimilarityTransformEstimator(py::module& m) {
                         "num_inliers"_a = report.support.num_inliers,
                         "inliers"_a = ToPythonMask(report.inlier_mask));
```

Comment:
Should this be `inlier_mask` to be consistent with the other changes?

Reply:
Good catch. Done.